packages/shell-api/src/replica-set.spec.ts

```diff
@@ -710,6 +710,9 @@ describe('ReplicaSet', () => {
     });
 
     describe('add member', () => {
+      // TODO: Fix these tests? They are currently failing with
```

Comment:
Oh interesting, just with 4.4? We could make the tests wait and retry.

Reply:
I think with 4.4 they still work … I’m not sure what the change in behavior is coming from, maybe we could just open a ticket about this after this is merged?

Reply:
SGTM